FusionMCPBridge/core/loader.py

```
@@ -0,0 +1,489 @@
+        Returns:
+            Number of modules successfully loaded
+        """
+        modules = self.discover_modules()
+        loaded_count = 0
+        
+        for module_name in modules:
+            if category.lower() in module_name.lower():
+                if self.load_module(module_name):
+                    loaded_count += 1
+        
+        logger.info(f"Loaded {loaded_count} modules in category: {category}")
```

Ruff (0.14.10)

Undefined name `logger`

(F821)